src/client/caching/stores/GuildMemberRoleStore.ts

```diff
@@ -0,0 +1,50 @@
+import { ProxyCache } from '@klasa/cache';
+import { Routes } from '@klasa/rest';
+
+import type { GuildMember } from '../structures/guilds/GuildMember';
+import type { Role } from '../structures/guilds/Role';
+import type { Guild } from '../structures/guilds/Guild';
+import type { Client } from '../../Client';
+
+export class GuildMemberRoleStore extends ProxyCache<string, Role> {
```

Comment:
We also need set support

Reply:
wdym?

Reply:
Design problem: `set` already exists (`ProxyCache#set`, which implements Map#set).

What would be a better replacement name? `edit`?

Reply:
Fixed in 1406fc6 :tada:!